tv_apps/browser/js/sync/settings.js

```diff
@@ -165,7 +165,7 @@
           // but we only want to do that if its enabled via user action
           // (and not because it is already enabled from a previous run).
           if (this.state === 'enabling') {
-            Settings.show();
+            Settings.show.bind(Settings)();
```

Comment:
Comparing to the functionality, the changed code is identical to the original code. Tried to reproduce without this patch but don't know what steps trigger the error. Does this line make a difference?